model/gcp/registry.yaml

```
@@ -195,3 +195,41 @@ groups:
         brief: >
           The hostname of a GCE instance. This is the full value of the default or [custom hostname](https://cloud.google.com/compute/docs/instances/custom-hostname-vm).
         examples: ['my-host1234.example.com', 'sample-vm.us-west1-b.c.my-project.internal']
+
+  - id: registry.gcp.gen_ai
+    type: attribute_group
+    display_name: Generative AI Attributes
+    brief: >
+      This documents defines attributes for Google generative AI systems.
+    stability: development
+    attributes:
+      - id: gcp.gen_ai.operation.config
+        stability: development
+        type: string
+        brief: Operation-specific request configuration options for Google generative AI operations recorded as a JSON string.
+        examples:
+         - '{"audioTimestamp":true}'
+         - '{"safetySettings":[{"threshold":"OFF"}]}'
+         - '{"thinkingConfig":{"includeThoughts":true},"routingConfig":{"autoMode":{"modelRoutingPreference":"BALANCED"}}}'
+        note: >
+          The data structure is dependent on the value of "gen_ai.operation.name" and should correspond
+          with the "*Config" data structure in the Google Gen AI SDK. The Google Gen AI SDK provides
+          these "*Config" data structures in multiple languages, generated from a common source of truth.
+          See "google.genai.types.*" in <https://github.com/googleapis/python-genai> and 
+          `com.google.genai.types.*` in <https://github.com/googleapis/java-genai> for examples.
+
+          The JSON dictionary SHOULD include only an allowlisted subset of the request configuration. The
+          contents also SHOULD NOT overlap with configuration that has already been captured elsewhere through
+          standard Semantic Conventions.
+
+          Because the data structure is encoded in JSON format, the key names should be specified
+          using "camelCase". More generally, the names should align with the expectations of proto3
+          JSON encoding <https://protobuf.dev/programming-guides/proto3/#json>; for example, fields
```

nit, maybe 

```suggestion
          The names SHOULD align with the expectations of proto3
          JSON encoding <https://protobuf.dev/programming-guides/proto3/#json>; for example, fields
```

for brevity?